servicemesh-install/topics/install-prerequisites.adoc

```diff
@@ -16,9 +16,11 @@ Before you can install the {ProductShortName} into an {product-title} installati
 [[updating-master-configuration]]
 == Updating the master configuration
 
+Service mesh relies on the existence of a proxy sidecar within the application's pod to provide service mesh capabilities to the application.
+
 [NOTE]
 ====
-The community version of Istio will inject the sidecar by default if you have labeled the namespace.  You are not required to label the namespace with {ProductName}. However, {ProductName} requires you to opt-in to having the sidecar automatically injected to a deployment.  This is to avoid injecting a sidecar where it is not wanted (for example build or deploy pods).
+ The community version of Istio injects the sidecar by default if you have labeled the namespace. You are not required to label the namespace with {ProductName}. However, {ProductName} requires you to opt-in to having the sidecar automatically injected to a deployment. This is to avoid injecting a sidecar where it is not wanted (for example build or deploy pods). We recommend automatic injection, using the annotation with no need to label namespaces, to ensure that your application contains the appropriate configuration for your service mesh upon deployment. This method requires fewer privileges and does not conflict with other OpenShift capabilities such as builder pods.
```

Comment:
When you say community here, would a user interpret this as upstream or Maistra? 

Does it make more sense to move the statement about how we have two supported methods of sidecar injection and that we suggest automatic  into the paragraph above instead of being in the note?